build/linux/installer/scripts/tomlparser-health-config.rb

```diff
@@ -13,32 +13,83 @@
 @configMapMountPath = "/etc/config/settings/agent-settings"
 @configSchemaVersion = ""
 @enable_health_model = false
+@nodesChunkSize = 0
+@podsChunkSize = 0
+@eventsChunkSize = 0
+@deploymentsChunkSize = 0
+@hpaChunkSize = 0
+@podsEmitStreamBatchSize = 0
```

Comment:
can chunk & emit size be same ? Whats the difference in perf we see for default pods (500 chunk, 200 emit, v 500 chunk. 500 emit) ?

Reply:
Yes, chunk and emit can be same size. Emit size controls the size of the file write and where as chunk size helps on the memory usage.  Based on the testing lower chunk size gives lower memory usage but that costs additional round trip latency.   To keep the ingestion latencies under 1 min, default value of pod chunk is 1K and pod emit size 200 items. Added the logic to validate the lower and upper bounds of these values and also added comments around this.